# -*- coding: utf-8 -*-
query = "Explore secreted CLEC11A protein in metastasis: current understanding of mechanisms, diagnostic potential, and therapeutic targeting opportunities."

plan = {
    "thought": "The user is asking about the role of secreted CLEC11A protein in metastasis, including its mechanisms, diagnostic potential, and therapeutic targeting opportunities. To create a logical chain, I need to start with understanding what CLEC11A is and establish its connection to metastasis, then explore the underlying mechanisms, diagnostic applications, and finally therapeutic targeting. This approach provides a comprehensive analysis from basic biology to clinical applications. The chain should be evidence-based and follow a natural progression from molecular mechanisms to clinical applications.",
    "Logical_chain": "CLEC11A structure and function → CLEC11A secretion and expression in cancer → CLEC11A's role in metastatic processes → Molecular mechanisms of CLEC11A in metastasis → CLEC11A as a biomarker for metastasis → Therapeutic targeting of CLEC11A in metastatic cancer",
    "steps": [
        {
            "relationship": "Starting point",
            "purpose": "To establish the basic understanding of CLEC11A protein structure, function, and normal physiological role",
            "Verification_Point": "Characterization of CLEC11A protein structure, cellular expression patterns, and physiological functions",
            "Step_query": "CLEC11A AND (protein structure OR function OR physiology)"
        },
        {
            "relationship": "Progression",
            "purpose": "To determine how CLEC11A is secreted and its expression patterns in various cancer types",
            "Verification_Point": "Evidence of CLEC11A secretion mechanisms and altered expression in cancer tissues compared to normal tissues",
            "Step_query": "CLEC11A AND (secretion OR expression) AND cancer"
        },
        {
            "relationship": "Causal relationship",
            "purpose": "To establish the connection between CLEC11A and key metastatic processes such as invasion, migration, and colonization",
            "Verification_Point": "Evidence linking CLEC11A to metastatic processes including cell migration, invasion, extravasation, and colonization",
            "Step_query": "CLEC11A AND (metastasis OR invasion OR migration OR colonization)"
        },
        {
            "relationship": "Mechanistic explanation",
            "purpose": "To understand the molecular mechanisms and signaling pathways through which CLEC11A promotes metastasis",
            "Verification_Point": "Molecular pathways, signaling cascades, and cellular interactions mediated by CLEC11A that contribute to metastasis",
            "Step_query": "CLEC11A AND (signaling pathway OR molecular mechanism) AND metastasis"
        },
        {
            "relationship": "Clinical application",
            "purpose": "To evaluate the potential of CLEC11A as a diagnostic or prognostic biomarker for metastatic cancer",
            "Verification_Point": "Studies assessing CLEC11A as a biomarker for detecting metastasis, monitoring disease progression, or predicting patient outcomes",
            "Step_query": "CLEC11A AND (biomarker OR diagnostic) AND (metastasis OR cancer)"
        },
        {
            "relationship": "Therapeutic implication",
            "purpose": "To explore strategies for therapeutic targeting of CLEC11A to prevent or treat metastasis",
            "Verification_Point": "Approaches for targeting CLEC11A including antibodies, small molecules, or other therapeutic modalities, and their efficacy in preclinical or clinical studies",
            "Step_query": "CLEC11A AND (therapeutic target OR inhibitor OR antibody) AND (metastasis OR cancer treatment)"
        }
    ]
}

paper = [{'biorxiv': [], 'medrxiv': [], 'pubmed': [
    {'Title': 'Identification of a mesenchymal progenitor cell hierarchy in adipose tissue.',
     'Abstract': 'Metabolic health depends on the capacity of adipose tissue progenitor cells to undergo de novo adipogenesis. The cellular hierarchy and mechanisms governing adipocyte progenitor differentiation are incompletely understood. Through single-cell RNA sequence analyses, we show that the lineage hierarchy of adipocyte progenitors consists of distinct mesenchymal cell types that are present in both mouse and human adipose tissues. Cells marked by dipeptidyl peptidase-4 (DPP4)/CD26 expression are highly proliferative, multipotent progenitors. During the development of subcutaneous adipose tissue in mice, these progenitor cells give rise to intercellular adhesion molecule-1 (ICAM1)/CD54-expressing (CD54+) committed preadipocytes and a related adipogenic cell population marked by <i>Clec11a</i> and <i>F3</i>/CD142 expression. Transforming growth factor-β maintains DPP4+ cell identity and inhibits adipogenic commitment of DPP4+ and CD142+ cells. Notably, DPP4+ progenitors reside in the reticular interstitium, a recently appreciated fluid-filled space within and between tissues, including adipose depots.',
     'PMID': '31023895', 'DOI': '10.1126/science.aav2501', 'Journal_Title': 'Science (New York, N.Y.)',
     'ISO_Abbreviation': 'Science', 'Journal_Score': '45.0', 'ISSN': '1095-9203', 'ArticleDate': '2019-04-26',
     'URL': 'https://pubmed.ncbi.nlm.nih.gov/31023895/', 'Level': 1}, {
        'Title': 'Single-cell analysis reveals altered tumor microenvironments of relapse- and remission-associated pediatric acute myeloid leukemia.',
        'Abstract': 'Acute myeloid leukemia (AML) microenvironment exhibits cellular and molecular differences among various subtypes. Here, we utilize single-cell RNA sequencing (scRNA-seq) to analyze pediatric AML bone marrow (BM) samples from diagnosis (Dx), end of induction (EOI), and relapse timepoints. Analysis of Dx, EOI scRNA-seq, and TARGET AML RNA-seq datasets reveals an AML blasts-associated 7-gene signature (CLEC11A, PRAME, AZU1, NREP, ARMH1, C1QBP, TRH), which we validate on independent datasets. The analysis reveals distinct clusters of Dx relapse- and continuous complete remission (CCR)-associated AML-blasts with differential expression of genes associated with survival. At Dx, relapse-associated samples have more exhausted T cells while CCR-associated samples have more inflammatory M1 macrophages. Post-therapy EOI residual blasts overexpress fatty acid oxidation, tumor growth, and stemness genes. Also, a post-therapy T-cell cluster associated with relapse samples exhibits downregulation of MHC Class I and T-cell regulatory genes. Altogether, this study deeply characterizes pediatric AML relapse- and CCR-associated samples to provide insights into the BM microenvironment landscape.',
        'PMID': '37798266', 'DOI': '10.1038/s41467-023-41994-0', 'Journal_Title': 'Nature communications',
        'ISO_Abbreviation': 'Nat Commun', 'Journal_Score': '13.5', 'ISSN': '2041-1723', 'ArticleDate': '2023-10-05',
        'URL': 'https://pubmed.ncbi.nlm.nih.gov/37798266/', 'Level': 2},
    {'Title': 'Molecular structure, expression, and functional role of Clec11a in skeletal biology and cancers.',
     'Abstract': 'C-type lectin domain family 11 member A (Clec11a), also known as stem cell growth factor (SCGF), C-type lectin superfamily member 3 (CLECSF3), or osteolectin was initially identified as a growth factor for hematopoietic progenitor cells. The human Clec11a gene encodes a polypeptide of 323 amino acids with characteristics of a secreted glycoprotein encompassing two integrin-binding motifs, RGD (Arg-Gly-Asp) and LDT (Leu-Asp-Thr), a putative leucine zipper\xa0domain, and a functional C-type lectin domain. It regulates hematopoietic differentiation and homeostasis\xa0and exhibits a protective effect against severe malarial anemia and lipotoxicity. Furthermore, Clec11a promotes the differentiation of mesenchymal progenitors into mature osteoblasts in vitro and plays an important role in the maintenance of adult skeleton age-related bone loss and fracture repair. Receptor ligand binding results in activation of downstream signaling cascades including glycogen synthase kinase 3 (GSK3), β-catenin, and Wnt, resulting in the expression of osteoblast-related gene transcripts including Alp, Runx2, Lef1, and Axin2. In addition, Clec11a is also associated with the development of several cancers, including leukemia, multiple myeloma, and gastrointestinal tract tumors. To date, however, the mechanisms governing transcription regulation of the Clec11a gene are\xa0not known and remain\xa0to be uncovered. Understanding the function and mechanism of action of Clec11a will pave the way for the development of Clec11a as a novel therapeutic target for conditions such as cancer, anemia, and skeletal diseases.',
     'PMID': '32003015', 'DOI': '10.1002/jcp.29600', 'Journal_Title': 'Journal of cellular physiology',
     'ISO_Abbreviation': 'J Cell Physiol', 'Journal_Score': '6.21428571428571', 'ISSN': '1097-4652',
     'ArticleDate': '2020-01-31', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/32003015/', 'Level': 1},
    {'Title': 'A mechanosensitive peri-arteriolar niche for osteogenesis and lymphopoiesis.',
     'Abstract': 'Stromal cells in adult bone marrow that express leptin receptor (LEPR) are a critical source of growth factors, including stem cell factor (SCF), for the maintenance of haematopoietic stem cells and early restricted progenitors<sup>1-6</sup>. LEPR<sup>+</sup> cells are heterogeneous, including skeletal stem cells and osteogenic and adipogenic progenitors<sup>7-12</sup>, although few markers have been available to distinguish these subsets or to compare their functions. Here we show that expression of an osteogenic growth factor, osteolectin<sup>13,14</sup>, distinguishes peri-arteriolar LEPR<sup>+</sup> cells poised to undergo osteogenesis from peri-sinusoidal LEPR<sup>+</sup> cells poised to undergo adipogenesis (but retaining osteogenic potential). Peri-arteriolar LEPR<sup>+</sup>osteolectin<sup>+</sup> cells are rapidly dividing, short-lived osteogenic progenitors that increase in number after fracture and are depleted during ageing. Deletion of Scf from adult osteolectin<sup>+</sup> cells did not affect the maintenance of haematopoietic stem cells or most restricted progenitors but depleted common lymphoid progenitors, impairing lymphopoiesis, bacterial clearance, and survival after acute bacterial infection. Peri-arteriolar osteolectin<sup>+</sup> cell maintenance required mechanical stimulation. Voluntary running increased, whereas hindlimb unloading decreased, the frequencies of peri-arteriolar osteolectin<sup>+</sup> cells and common lymphoid progenitors. Deletion of the mechanosensitive ion channel PIEZO1 from osteolectin<sup>+</sup> cells depleted osteolectin<sup>+</sup> cells and common lymphoid progenitors. These results show that a peri-arteriolar niche for osteogenesis and lymphopoiesis in bone marrow is maintained by mechanical stimulation and depleted during ageing.',
     'PMID': '33627868', 'DOI': '10.1038/s41586-021-03298-5', 'Journal_Title': 'Nature', 'ISO_Abbreviation': 'Nature',
     'Journal_Score': '67.5', 'ISSN': '1476-4687', 'ArticleDate': '2021-02-24',
     'URL': 'https://pubmed.ncbi.nlm.nih.gov/33627868/', 'Level': 2},
    {'Title': 'Identification and characterization of CLEC11A and its derived immune signature in gastric cancer.',
     'Abstract': 'C-type lectin domain family 11 member A (CLEC11A) was characterized as a growth factor that mainly regulates hematopoietic function and differentiation of bone cells. However, the involvement of CLEC11A in gastric cancer (GC) is not well understood.',
     'PMID': '38348052', 'DOI': '10.3389/fimmu.2024.1324959', 'Journal_Title': 'Frontiers in immunology',
     'ISO_Abbreviation': 'Front Immunol', 'Journal_Score': '5.71875', 'ISSN': '1664-3224', 'ArticleDate': '2024-01-29',
     'URL': 'https://pubmed.ncbi.nlm.nih.gov/38348052/', 'Level': 1},
    {'Title': 'The role of Clec11a in bone construction and remodeling.',
     'Abstract': 'Bone is a dynamically active tissue whose health status is closely related to its construction and remodeling, and imbalances in bone homeostasis lead to a wide range of bone diseases. The sulfated glycoprotein C-type lectin structural domain family 11 member A (Clec11a) is a key factor in bone mass regulation that significantly promotes the osteogenic differentiation of bone marrow mesenchymal stem cells and osteoblasts and stimulates chondrocyte proliferation, thereby promoting longitudinal bone growth. More importantly, Clec11a has high therapeutic potential for treating various bone diseases and can enhance the therapeutic effects of the parathyroid hormone against osteoporosis. Clec11a is also involved in the stress/adaptive response of bone to exercise via mechanical stimulation of the cation channel Pieoz1. Clec11a plays an important role in promoting bone health and preventing bone disease and may represent a new target and novel drug for bone disease treatment. Therefore, this review aims to explore the role and possible mechanisms of Clec11a in the skeletal system, evaluate its value as a potential therapeutic target against bone diseases, and provide new ideas and strategies for basic research on Clec11a and preventing and treating bone disease.',
     'PMID': '39188913', 'DOI': '10.3389/fendo.2024.1429567', 'Journal_Title': 'Frontiers in endocrinology',
     'ISO_Abbreviation': 'Front Endocrinol (Lausanne)', 'Journal_Score': '4.44186046511628', 'ISSN': '1664-2392',
     'ArticleDate': '2024-08-12', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/39188913/', 'Level': 2}, {
        'Title': 'Human umbilical cord mesenchymal stromal cells-derived extracellular vesicles exert potent bone protective effects by CLEC11A-mediated regulation of bone metabolism.',
        'Abstract': 'Osteoporosis and osteoporotic fractures severely compromise quality of life in elderly people and lead to early death. Human umbilical cord mesenchymal stromal cell (MSC)-derived extracellular vesicles (hucMSC-EVs) possess considerable therapeutic effects in tissue repair and regeneration. Thus, in the present study, we investigated the effects of hucMSC-EVs on primary and secondary osteoporosis and explored the underlying mechanisms. <b>Methods</b>: hucMSCs were isolated and cultured. EVs were obtained from the conditioned medium of hucMSCs and determined by using transmission electron microscopy, dynamic light scattering and Western Blot analyses. The effects of hucMSC-EVs on ovariectomy-induced postmenopausal osteoporosis and tail suspension-induced hindlimb disuse osteoporosis in mouse models were assessed by using microcomputed tomography, biomechanical, histochemical and immunohistochemical, as well as histomorphometric analyses. Proteomic analysis was applied between hucMSC-EVs and hucMSCs to screen the candidate proteins that mediate hucMSC-EVs function. The effects of hucMSC-EVs on osteogenic and adipogenic differentiation of bone marrow mesenchymal stromal cells (BMSCs), and osteoclastogenesis of the macrophage cell line RAW264.7 <i>in vitro</i> were determined by using cytochemical staining and quantitative real-time PCR analysis. Subsequently, the roles of the key protein in hucMSC-EVs-induced regulation on BMSCs and RAW264.7 cells were evaluated. <b>Results:</b> hucMSCs were able to differentiate into osteoblasts, adipocytes or chondrocytes and positively expressed CD29, CD44, CD73 and CD90, but negatively expressed CD34 and CD45. The morphological assessment revealed the typical cup- or sphere-shaped morphology of hucMSC-EVs with diameters predominantly ranging from 60 nm to 150 nm and expressed CD9, CD63, CD81 and TSG101. The systemic administration of hucMSC-EVs prevented bone loss and maintained bone strength in osteoporotic mice by enhancing bone formation, reducing marrow fat accumulation and decreasing bone resorption. Proteomic analysis showed that the potently pro-osteogenic protein, CLEC11A (C-type lectin domain family 11, member A) was very highly enriched in hucMSC-EVs. In addition, hucMSC-EVs enhanced the shift from adipogenic to osteogenic differentiation of BMSCs via delivering CLEC11A <i>in vitro</i>. Moreover, CLEC11A was required for the inhibitory effects of hucMSC-EVs on osteoclast formation. <b>Conclusion:</b> Our results suggest that hucMSC-EVs serve as a critical regulator of bone metabolism by transferring CLEC11A and may represent a potential agent for prevention and treatment of osteoporosis.',
        'PMID': '32089743', 'DOI': '10.7150/thno.39238', 'Journal_Title': 'Theranostics',
        'ISO_Abbreviation': 'Theranostics', 'Journal_Score': '27.8571428571429', 'ISSN': '1838-7640',
        'ArticleDate': '2020-01-16', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/32089743/', 'Level': 2},
    {'Title': 'CLEC11A improves insulin secretion and promotes cell proliferation in human beta-cells.',
     'Abstract': 'Beta-cell dysfunction is a hallmark of disease progression in patients with diabetes. Research has been focused on maintaining and restoring beta-cell function during diabetes development. The aims of this study were to explore the expression of C-type lectin domain containing 11A (CLEC11A), a secreted sulphated glycoprotein, in human islets and to evaluate the effects of CLEC11A on beta-cell function and proliferation in vitro. To test these hypotheses, human islets and human EndoC-βH1 cell line were used in this study. We identified that CLEC11A was expressed in beta-cells and alpha-cells in human islets but not in EndoC-βH1 cells, whereas the receptor of CLEC11A called integrin subunit alpha 11 was found in both human islets and EndoC-βH1 cells. Long-term treatment with exogenous recombinant human CLEC11A (rhCLEC11A) accentuated glucose-stimulated insulin secretion, insulin content, and proliferation from human islets and EndoC-βH1 cells, which was partially due to the accentuated expression levels of transcription factors MAFA and PDX1. However, the impaired beta-cell function and reduced mRNA expression of INS and MAFA in EndoC-βH1 cells that were caused by chronic palmitate exposure could only be partially improved by the introduction of rhCLEC11A. Based on these results, we conclude that rhCLEC11A promotes insulin secretion, insulin content, and proliferation in human beta-cells, which are associated with the accentuated expression levels of transcription factors MAFA and PDX1. CLEC11A, therefore, may provide a novel therapeutic target for maintaining beta-cell function in patients with diabetes.',
     'PMID': '37078556', 'DOI': '10.1530/JME-22-0066', 'Journal_Title': 'Journal of molecular endocrinology',
     'ISO_Abbreviation': 'J Mol Endocrinol', 'Journal_Score': '3.18333333333333', 'ISSN': '1479-6813',
     'ArticleDate': '2023-06-21', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/37078556/', 'Level': 2},
    {'Title': 'Clec11a/osteolectin is an osteogenic growth factor that promotes the maintenance of the adult skeleton.',
     'Abstract': 'Bone marrow stromal cells maintain the adult skeleton by forming osteoblasts throughout life that regenerate bone and repair fractures. We discovered that subsets of these stromal cells, osteoblasts, osteocytes, and hypertrophic chondrocytes secrete a C-type lectin domain protein, Clec11a, which promotes osteogenesis. <i>Clec11a</i>-deficient mice appeared developmentally normal and had normal hematopoiesis but reduced limb and vertebral bone. <i>Clec11a</i>-deficient mice exhibited accelerated bone loss during aging, reduced bone strength, and delayed fracture healing. Bone marrow stromal cells from <i>Clec11a</i>-deficient mice showed impaired osteogenic differentiation, but normal adipogenic and chondrogenic differentiation. Recombinant Clec11a promoted osteogenesis by stromal cells in culture and increased bone mass in osteoporotic mice in vivo. Recombinant human Clec11a promoted osteogenesis by human bone marrow stromal cells in culture and in vivo. Clec11a thus maintains the adult skeleton by promoting the differentiation of mesenchymal progenitors into mature osteoblasts. In light of this, we propose to call this factor Osteolectin.',
     'PMID': '27976999', 'DOI': '10.7554/eLife.18782', 'Journal_Title': 'eLife', 'ISO_Abbreviation': 'Elife',
     'Journal_Score': 'nan', 'ISSN': '2050-084X', 'ArticleDate': '2016-12-13',
     'URL': 'https://pubmed.ncbi.nlm.nih.gov/27976999/', 'Level': 2}]}, {'biorxiv': [], 'medrxiv': [], 'pubmed': [
    {'Title': 'Identification and characterization of CLEC11A and its derived immune signature in gastric cancer.',
     'Abstract': 'C-type lectin domain family 11 member A (CLEC11A) was characterized as a growth factor that mainly regulates hematopoietic function and differentiation of bone cells. However, the involvement of CLEC11A in gastric cancer (GC) is not well understood.',
     'PMID': '38348052', 'DOI': '10.3389/fimmu.2024.1324959', 'Journal_Title': 'Frontiers in immunology',
     'ISO_Abbreviation': 'Front Immunol', 'Journal_Score': '5.71875', 'ISSN': '1664-3224', 'ArticleDate': '2024-01-29',
     'URL': 'https://pubmed.ncbi.nlm.nih.gov/38348052/', 'Level': 1}, {
        'Title': 'Single-cell analysis reveals altered tumor microenvironments of relapse- and remission-associated pediatric acute myeloid leukemia.',
        'Abstract': 'Acute myeloid leukemia (AML) microenvironment exhibits cellular and molecular differences among various subtypes. Here, we utilize single-cell RNA sequencing (scRNA-seq) to analyze pediatric AML bone marrow (BM) samples from diagnosis (Dx), end of induction (EOI), and relapse timepoints. Analysis of Dx, EOI scRNA-seq, and TARGET AML RNA-seq datasets reveals an AML blasts-associated 7-gene signature (CLEC11A, PRAME, AZU1, NREP, ARMH1, C1QBP, TRH), which we validate on independent datasets. The analysis reveals distinct clusters of Dx relapse- and continuous complete remission (CCR)-associated AML-blasts with differential expression of genes associated with survival. At Dx, relapse-associated samples have more exhausted T cells while CCR-associated samples have more inflammatory M1 macrophages. Post-therapy EOI residual blasts overexpress fatty acid oxidation, tumor growth, and stemness genes. Also, a post-therapy T-cell cluster associated with relapse samples exhibits downregulation of MHC Class I and T-cell regulatory genes. Altogether, this study deeply characterizes pediatric AML relapse- and CCR-associated samples to provide insights into the BM microenvironment landscape.',
        'PMID': '37798266', 'DOI': '10.1038/s41467-023-41994-0', 'Journal_Title': 'Nature communications',
        'ISO_Abbreviation': 'Nat Commun', 'Journal_Score': '13.5', 'ISSN': '2041-1723', 'ArticleDate': '2023-10-05',
        'URL': 'https://pubmed.ncbi.nlm.nih.gov/37798266/', 'Level': 2},
    {'Title': 'Molecular structure, expression, and functional role of Clec11a in skeletal biology and cancers.',
     'Abstract': 'C-type lectin domain family 11 member A (Clec11a), also known as stem cell growth factor (SCGF), C-type lectin superfamily member 3 (CLECSF3), or osteolectin was initially identified as a growth factor for hematopoietic progenitor cells. The human Clec11a gene encodes a polypeptide of 323 amino acids with characteristics of a secreted glycoprotein encompassing two integrin-binding motifs, RGD (Arg-Gly-Asp) and LDT (Leu-Asp-Thr), a putative leucine zipper\xa0domain, and a functional C-type lectin domain. It regulates hematopoietic differentiation and homeostasis\xa0and exhibits a protective effect against severe malarial anemia and lipotoxicity. Furthermore, Clec11a promotes the differentiation of mesenchymal progenitors into mature osteoblasts in vitro and plays an important role in the maintenance of adult skeleton age-related bone loss and fracture repair. Receptor ligand binding results in activation of downstream signaling cascades including glycogen synthase kinase 3 (GSK3), β-catenin, and Wnt, resulting in the expression of osteoblast-related gene transcripts including Alp, Runx2, Lef1, and Axin2. In addition, Clec11a is also associated with the development of several cancers, including leukemia, multiple myeloma, and gastrointestinal tract tumors. To date, however, the mechanisms governing transcription regulation of the Clec11a gene are\xa0not known and remain\xa0to be uncovered. Understanding the function and mechanism of action of Clec11a will pave the way for the development of Clec11a as a novel therapeutic target for conditions such as cancer, anemia, and skeletal diseases.',
     'PMID': '32003015', 'DOI': '10.1002/jcp.29600', 'Journal_Title': 'Journal of cellular physiology',
     'ISO_Abbreviation': 'J Cell Physiol', 'Journal_Score': '6.21428571428571', 'ISSN': '1097-4652',
     'ArticleDate': '2020-01-31', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/32003015/', 'Level': 1},
    {'Title': 'CLEC11A expression as a prognostic biomarker in correlation to immune cells of gastric cancer.',
     'Abstract': "Gastric cancer (GC) is a prevalent malignant cancer characterized by a poor survival rate. The C-type lectin domain family 11 member A (CLEC11A) is part of the C-type lectin superfamily, and its dysregulation has been implicated in the progression of several cancers. The specific role of CLEC11A and its association with immune infiltration in GC, however, remains unclear. In this study, we employed The Cancer Genome Atlas (TCGA) database, Gene Expression Omnibus (GEO) database, Tumor IMmune Estimation Resource (TIMER) database, Gene Expression Profiling Interactive Analysis (GEPIA), UALCAN, Kaplan-Meier plotter databases, gene ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), and the CIBERSORT algorithm to investigate CLEC11A expression, its prognostic significance, its association with tumor immune infiltration, and gene function enrichment in GC. We conducted western blotting, Cell Counting Kit-8 (CCK-8), wound healing, and transwell assays to validate CLEC11A's function. We found that CLEC11A expression was significantly elevated in GC when compared to adjacent non-tumor tissues. Elevated CLEC11A expression was strongly associated with poor survival outcomes and advanced clinicopathological stages. \xa0Moreover, heightened CLEC11A expression positively correlated with immunomodulators, chemokines, and the infiltration of immune cells, especially M2 macrophages, in GC. Additionally, CLEC11A silencing suppressed GC cells proliferation, migration and invasion in vitro. Our results elucidate the functions of CLEC11A in GC, suggesting its potential as a valuable prognostic biomarker and therapeutic target for GC immunotherapy.",
     'PMID': '37597212', 'DOI': '10.17305/bb.2023.9384', 'Journal_Title': 'Biomolecules & biomedicine',
     'ISO_Abbreviation': 'Biomol Biomed', 'Journal_Score': '1.78899082568807', 'ISSN': '2831-090X',
     'ArticleDate': '2024-01-03', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/37597212/', 'Level': 1}, {
        'Title': 'Characterization of the tumour microenvironment phenotypes in malignant tissues and pleural effusion from advanced osteoblastic osteosarcoma patients.',
        'Abstract': 'Malignant pleural effusion (MPE) is an adverse prognostic factor in patients with osteoblastic osteosarcoma; however, the cellular contexts of MPE are largely unknown.',
        'PMID': '36305631', 'DOI': '10.1002/ctm2.1072', 'Journal_Title': 'Clinical and translational medicine',
        'ISO_Abbreviation': 'Clin Transl Med', 'Journal_Score': '6.93617021276596', 'ISSN': '2001-1326',
        'ArticleDate': '2022-11', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/36305631/', 'Level': 2},
    {'Title': 'High Expression of CLEC11A Predicts Favorable Prognosis in Acute Myeloid Leukemia.',
     'Abstract': 'Acute myeloid leukemia (AML) is a heterogeneous disease of the hematopoietic system, for which identification of novel molecular\xa0markers is potentially important for clinical prognosis and is an urgent need for treatment optimization.',
     'PMID': '33747924', 'DOI': '10.3389/fonc.2021.608932', 'Journal_Title': 'Frontiers in oncology',
     'ISO_Abbreviation': 'Front Oncol', 'Journal_Score': '2.65040650406504', 'ISSN': '2234-943X',
     'ArticleDate': '2021-03-02', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/33747924/', 'Level': 2}, {
        'Title': 'CLEC11A methylation is correlated to AML subtypes and cytogenetic risk factors but not patient demographics.',
        'Abstract': 'Acute myeloid leukemia (AML) is an aggressive and lethal cancer of the blood, which leads to the death of over 11,000 patients in the United States each year. Research on identifying, characterizing, and treating AML is crucial in the fight against this deadly disease. Recent studies have examined the role of CLEC11A in cancer, including AML. However, there have been conflicting reports related to tumor progression and survival. Because survival is based on a variety of factors, including classification of the tumor, genetic risk factors, and demographics, it is imperative that we determine what role CLEC11A may have in cancer survival. Therefore, utilizing data from the Genomic Data Commons, we analyzed CLEC11A methylation in 108 AML patients compared to FAB classification, cytogenetic risk factors, age, race, and gender. Our results show statistically significant correlations between methylation of CLEC11A and FAB classification as well as poor genetic risk factors. However, no difference was observed in CLEC11A methylation when compared to demographic data. Our results, matched with a known biological function of CLEC11A in early hematopoiesis, indicate that CLEC11A may be an important marker for AML diagnosis and prognosis and provide relevant data in the ongoing search for novel therapeutics to improve AML survival.',
        'PMID': '38466706', 'DOI': '10.1371/journal.pone.0300477', 'Journal_Title': 'PloS one',
        'ISO_Abbreviation': 'PLoS One', 'Journal_Score': '3.06818181818182', 'ISSN': '1932-6203',
        'ArticleDate': '2024-03-11', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/38466706/', 'Level': 2},
    {'Title': 'Identification of a Cancer Stem Cells Signature of Head and Neck Squamous Cell Carcinoma.',
     'Abstract': '<b>Background:</b> Head and neck squamous cell carcinoma (HNSCC) ranks as the sixth most widespread and deadly cancer. In recent times, it has been determined that undifferentiated cell populations with stem cell-like properties in HNSCC are major factors influencing recurrence and progression. <b>Method:</b> In this study, we determine key genes related to stemness by merging WGCNA with HNSCC mRNAsi based on the online database. <b>Results:</b> We first download the mRNA expression-based stemness index (mRNAsi) data and contrast the expression levels of mRNAsi in cancers and control samples; we found significantly elevated mRNAsi expressions in HNSCC tissues (<i>p</i> = 0.002). Moreover, the brown module showed a relatively high negative correlation with mRNAsi (cor = -0.8). Thus, we selected the brown module as the interesting module and used it for following analysis. We screened 20 key genes (PDGFRB, PLPP4, CALU, ADAMTS14, COL5A3, KCNE4, LOXL1, CLEC11A, PODN,BGN, AEBP1, COL1A2, LAMA4, LOXL2, LRRC15, THY1, SPON2, COL1A1, NID2, and AC134312.5) including and as to decide the neighbor genes biological interaction network of these 20 stemness-related genes in HNSCC. The top 10 frequent alterations were PIK3CA, FGF3, FGF19, FGF4, DVL3, P3H2, GNB4, COL22A1, COL14A1, and PLOD2. <b>Conclusion:</b> This study showed the critical role of stemness-related genes in HNSCC. However, more related studies are needed to confirm these results.',
     'PMID': '35646104', 'DOI': '10.3389/fgene.2022.814777', 'Journal_Title': 'Frontiers in genetics',
     'ISO_Abbreviation': 'Front Genet', 'Journal_Score': '2.41772151898734', 'ISSN': '1664-8021',
     'ArticleDate': '2022-05-11', 'URL': 'https://pubmed.ncbi.nlm.nih.gov/35646104/', 'Level': 2}]},
         {'biorxiv': [], 'medrxiv': [], 'pubmed': [{
                                                       'Title': 'Identification and characterization of CLEC11A and its derived immune signature in gastric cancer.',
                                                       'Abstract': 'C-type lectin domain family 11 member A (CLEC11A) was characterized as a growth factor that mainly regulates hematopoietic function and differentiation of bone cells. However, the involvement of CLEC11A in gastric cancer (GC) is not well understood.',
                                                       'PMID': '38348052', 'DOI': '10.3389/fimmu.2024.1324959',
                                                       'Journal_Title': 'Frontiers in immunology',
                                                       'ISO_Abbreviation': 'Front Immunol', 'Journal_Score': '5.71875',
                                                       'ISSN': '1664-3224', 'ArticleDate': '2024-01-29',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/38348052/', 'Level': 1},
                                                   {
                                                       'Title': 'Characterization of the tumour microenvironment phenotypes in malignant tissues and pleural effusion from advanced osteoblastic osteosarcoma patients.',
                                                       'Abstract': 'Malignant pleural effusion (MPE) is an adverse prognostic factor in patients with osteoblastic osteosarcoma; however, the cellular contexts of MPE are largely unknown.',
                                                       'PMID': '36305631', 'DOI': '10.1002/ctm2.1072',
                                                       'Journal_Title': 'Clinical and translational medicine',
                                                       'ISO_Abbreviation': 'Clin Transl Med',
                                                       'Journal_Score': '6.93617021276596', 'ISSN': '2001-1326',
                                                       'ArticleDate': '2022-11',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/36305631/', 'Level': 2},
                                                   {
                                                       'Title': 'CLEC11A expression as a prognostic biomarker in correlation to immune cells of gastric cancer.',
                                                       'Abstract': "Gastric cancer (GC) is a prevalent malignant cancer characterized by a poor survival rate. The C-type lectin domain family 11 member A (CLEC11A) is part of the C-type lectin superfamily, and its dysregulation has been implicated in the progression of several cancers. The specific role of CLEC11A and its association with immune infiltration in GC, however, remains unclear. In this study, we employed The Cancer Genome Atlas (TCGA) database, Gene Expression Omnibus (GEO) database, Tumor IMmune Estimation Resource (TIMER) database, Gene Expression Profiling Interactive Analysis (GEPIA), UALCAN, Kaplan-Meier plotter databases, gene ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), and the CIBERSORT algorithm to investigate CLEC11A expression, its prognostic significance, its association with tumor immune infiltration, and gene function enrichment in GC. We conducted western blotting, Cell Counting Kit-8 (CCK-8), wound healing, and transwell assays to validate CLEC11A's function. We found that CLEC11A expression was significantly elevated in GC when compared to adjacent non-tumor tissues. Elevated CLEC11A expression was strongly associated with poor survival outcomes and advanced clinicopathological stages. \xa0Moreover, heightened CLEC11A expression positively correlated with immunomodulators, chemokines, and the infiltration of immune cells, especially M2 macrophages, in GC. Additionally, CLEC11A silencing suppressed GC cells proliferation, migration and invasion in vitro. Our results elucidate the functions of CLEC11A in GC, suggesting its potential as a valuable prognostic biomarker and therapeutic target for GC immunotherapy.",
                                                       'PMID': '37597212', 'DOI': '10.17305/bb.2023.9384',
                                                       'Journal_Title': 'Biomolecules & biomedicine',
                                                       'ISO_Abbreviation': 'Biomol Biomed',
                                                       'Journal_Score': '1.78899082568807', 'ISSN': '2831-090X',
                                                       'ArticleDate': '2024-01-03',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/37597212/', 'Level': 1},
                                                   {
                                                       'Title': 'Integrating Machine Learning and the SHapley Additive exPlanations (SHAP) Framework to Predict Lymph Node Metastasis in Gastric Cancer Patients Based on Inflammation Indices and Peripheral Lymphocyte Subpopulations.',
                                                       'Abstract': 'The prediction of lymph node metastasis in gastric cancer, a pivotal determinant affecting treatment approaches and prognosis, continues to pose a significant challenge in terms of accuracy.',
                                                       'PMID': '39606641', 'DOI': '10.2147/JIR.S488676',
                                                       'Journal_Title': 'Journal of inflammation research',
                                                       'ISO_Abbreviation': 'J Inflamm Res',
                                                       'Journal_Score': '2.95161290322581', 'ISSN': '1178-7031',
                                                       'ArticleDate': '2024-11-23',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/39606641/',
                                                       'Level': 2}]}, {'biorxiv': [
        {'Title': 'Alzheimer-related pathogenesis is dependent on neuronal receptor PTPσ.',
         'Abstract': 'Due to limited understanding of disease mechanisms and the lack of molecular targets, translational research for Alzheimer disease has not been fruitful hitherto. Here we report findings that indicate neuronal receptor phosphatase PTP{sigma} as a potential therapeutic target for this dementia. In two TgAPP mouse models, a spectrum of Alzheimer-related pathologies, including aged-induced progression of {beta}-amyloidosis, Tau aggregation, neuroinflammation, synaptic loss, as well as behavioral deficits, all show unambiguous dependency on PTP{sigma}. APP amyloidogenic metabolites diminish upon PTP{sigma} genetic depletion or pharmacological inhibition. Binding to APP in the brain, PTP{sigma} regulates APP proteolytic metabolism via its phosphatase activity, likely through downstream signaling that modulates APP membrane localization and affinity to the {beta}-secretase, in a specific manner that does not broadly affect {beta}- and {gamma}-secretase processing of other major substrates. Together, these findings unveil a gatekeeping role of PTP{sigma} upstream in Alzheimer-like pathogenic pathway.',
         'URL': 'https://www.biorxiv.org/content/10.1101/079806v5', 'ArticleDate': '2024-04-04',
         'DOI': '10.1101/079806', 'Level': 2}], 'medrxiv': [{
                                                                'Title': "Fixel-based analysis reveals macrostructural white matter changes associated with tau pathology in early stages of Alzheimer's disease",
                                                                'Abstract': 'White matter (WM) alterations are commonly found across different stages of Alzheimers disease (AD). However, the association between these changes with underlying AD pathology such as amyloid-{beta} (A{beta}) and tau deposition is still poorly understood. Hitherto, most studies have assessed WM alterations in AD using diffusion tensor imaging (DTI). Nonetheless, DTI has methodological shortcomings that limit an accurate biological interpretation. To address this limitation, here we applied fixel-based analysis (FBA) to disentangle microscopic differences in fiber density (FD) from macroscopic morphological changes in fiber cross-section (FC) in early stages of AD. We further investigated the associations of FBA metrics with AD pathology and cognitive performance. Additionally, we compared FBA results with other commonly used WM metrics derived from DTI, free-water corrected (FW)-DTI and diffusion kurtosis imaging (DKI). To achieve these goals, we included 224 A{beta}-negative and 91 A{beta}-positive cognitively unimpaired individuals as well as 78 A{beta}-positive patients with mild cognitive impairment (MCI) with diffusion-weighted MRI (dMRI), A{beta}-PET and tau-PET scans from the Swedish BioFINDER-2 study. We found that tau-PET uptake in medial temporal regions was associated with macrostructural alterations reflected by reduced FC mainly in the parahippocampal part of the cingulum bundle in A{beta}-positive individuals. This tau-related WM alteration was also associated with impaired memory. Interestingly, only FBA metrics were able to capture the association between tau-PET uptake and white matter degeneration. No association was found between global amyloid load and any dMRI metrics. Compared to both cognitively unimpaired groups, MCI patients showed a decrease in all FBA metrics in the entire cingulum bundle, uncinate fasciculus and anterior thalamic radiations. Metrics derived from DKI, and FW-DTI revealed a similar pattern of alterations whereas the spatial extent of WM abnormalities detected by DTI was more widespread. Altogether, our results indicate that early WM alterations in AD are mainly due to macrostructural changes identified by FBA metrics, being more closely associated with tau than A{beta} pathology. These findings suggest that future studies assessing the effects of AD pathology in white matter tracts should consider using FBA metrics.',
                                                                'URL': 'https://www.medrxiv.org/content/10.1101/2023.02.17.23286094v1',
                                                                'ArticleDate': '2023-02-18',
                                                                'DOI': '10.1101/2023.02.17.23286094', 'Level': 2}],
                                                                       'pubmed': []}, {'biorxiv': [], 'medrxiv': [{
                                                                                                                      'Title': 'Re-weighting MC1R, ASIP and IRF4 risk variants optimises polygenic risk scores for keratinocyte cancer stratification in solid organ transplant recipients.',
                                                                                                                      'Abstract': 'IntroductionSolid organ transplant recipients (SOTRs) are at much higher risk of developing squamous cell carcinoma (SCC) and basal cell carcinoma (BCC), compared to the general population. Previous studies have derived genetics-based predictors (polygenic risk scores, PRS) of SCC and BCC risk in SOTRs by assuming that genetic risk variants act in the same way in the general population as in SOTRs, but this assumption has not been fully tested.\n\nObjectiveTo investigate whether known genetic risk variants for SCC and BCC have different effect sizes in SOTRs versus in non-transplantees, and if a re-weighted PRS would improve risk prediction.\n\nMethodsWe conducted genome-wide association studies for SCC and BCC separately in the non-transplant general population and in SOTRs, and compared the risks associated with selected common genetic variants for KC risk in SOTR vs non-transplant individuals from the UK Biobank. For regions with an increased log odds ratio in SOTRs, PRSs including these weights were validated in the QSkin study, and applied to the Australian STAR SOTR cohort.\n\nResultsEffect sizes for functional variants in MC1R (rs1805007), ASIP (rs6059655), and IRF4 (rs12203592) were much more strongly associated with the risk of KC in SOTRs than in non-transplantees. The proportional increase in the effect sizes ranged from 1.9-fold for rs6059655 and BCC risk (SOTRs log (OR)=0.49, 95%CI=0.00-0.98 vs log (OR)=0.26, 95%CI=0.24-0.30 in non-transplantees) to as high as 4.8-fold for rs1805007 and SCC risk (SOTR log (OR)=0.88, 95% CI=0.41-1.35 vs log (OR)=0.18, 95% CI=0.12-0.24 in non-transplantees). PRS with SOTR derived weights for these SNPs showed improved SCC/BCC risk stratification in the STAR Cohort, with the optimised PRS reclassifying 19% of SCC cases vs 8% using the standard PRS, and 18% of BCC cases vs 12% using the standard PRS.\n\nConclusionEffect sizes for SCC and BCC risk for genetic variants in the MC1R, ASIP and IRF4 genes are elevated in SOTRs, and correctly weighting these variants improves risk stratification based on polygenic risk.',
                                                                                                                      'URL': 'https://www.medrxiv.org/content/10.1101/2023.02.17.23286114v1',
                                                                                                                      'ArticleDate': '2023-02-18',
                                                                                                                      'DOI': '10.1101/2023.02.17.23286114',
                                                                                                                      'Level': 2}],
                                                                                       'pubmed': [{
                                                                                                      'Title': 'Single-cell analysis reveals altered tumor microenvironments of relapse- and remission-associated pediatric acute myeloid leukemia.',
                                                                                                      'Abstract': 'Acute myeloid leukemia (AML) microenvironment exhibits cellular and molecular differences among various subtypes. Here, we utilize single-cell RNA sequencing (scRNA-seq) to analyze pediatric AML bone marrow (BM) samples from diagnosis (Dx), end of induction (EOI), and relapse timepoints. Analysis of Dx, EOI scRNA-seq, and TARGET AML RNA-seq datasets reveals an AML blasts-associated 7-gene signature (CLEC11A, PRAME, AZU1, NREP, ARMH1, C1QBP, TRH), which we validate on independent datasets. The analysis reveals distinct clusters of Dx relapse- and continuous complete remission (CCR)-associated AML-blasts with differential expression of genes associated with survival. At Dx, relapse-associated samples have more exhausted T cells while CCR-associated samples have more inflammatory M1 macrophages. Post-therapy EOI residual blasts overexpress fatty acid oxidation, tumor growth, and stemness genes. Also, a post-therapy T-cell cluster associated with relapse samples exhibits downregulation of MHC Class I and T-cell regulatory genes. Altogether, this study deeply characterizes pediatric AML relapse- and CCR-associated samples to provide insights into the BM microenvironment landscape.',
                                                                                                      'PMID': '37798266',
                                                                                                      'DOI': '10.1038/s41467-023-41994-0',
                                                                                                      'Journal_Title': 'Nature communications',
                                                                                                      'ISO_Abbreviation': 'Nat Commun',
                                                                                                      'Journal_Score': '13.5',
                                                                                                      'ISSN': '2041-1723',
                                                                                                      'ArticleDate': '2023-10-05',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/37798266/',
                                                                                                      'Level': 1}, {
                                                                                                      'Title': 'Identification and characterization of CLEC11A and its derived immune signature in gastric cancer.',
                                                                                                      'Abstract': 'C-type lectin domain family 11 member A (CLEC11A) was characterized as a growth factor that mainly regulates hematopoietic function and differentiation of bone cells. However, the involvement of CLEC11A in gastric cancer (GC) is not well understood.',
                                                                                                      'PMID': '38348052',
                                                                                                      'DOI': '10.3389/fimmu.2024.1324959',
                                                                                                      'Journal_Title': 'Frontiers in immunology',
                                                                                                      'ISO_Abbreviation': 'Front Immunol',
                                                                                                      'Journal_Score': '5.71875',
                                                                                                      'ISSN': '1664-3224',
                                                                                                      'ArticleDate': '2024-01-29',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/38348052/',
                                                                                                      'Level': 1}, {
                                                                                                      'Title': 'CLEC11A expression as a prognostic biomarker in correlation to immune cells of gastric cancer.',
                                                                                                      'Abstract': "Gastric cancer (GC) is a prevalent malignant cancer characterized by a poor survival rate. The C-type lectin domain family 11 member A (CLEC11A) is part of the C-type lectin superfamily, and its dysregulation has been implicated in the progression of several cancers. The specific role of CLEC11A and its association with immune infiltration in GC, however, remains unclear. In this study, we employed The Cancer Genome Atlas (TCGA) database, Gene Expression Omnibus (GEO) database, Tumor IMmune Estimation Resource (TIMER) database, Gene Expression Profiling Interactive Analysis (GEPIA), UALCAN, Kaplan-Meier plotter databases, gene ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), and the CIBERSORT algorithm to investigate CLEC11A expression, its prognostic significance, its association with tumor immune infiltration, and gene function enrichment in GC. We conducted western blotting, Cell Counting Kit-8 (CCK-8), wound healing, and transwell assays to validate CLEC11A's function. We found that CLEC11A expression was significantly elevated in GC when compared to adjacent non-tumor tissues. Elevated CLEC11A expression was strongly associated with poor survival outcomes and advanced clinicopathological stages. \xa0Moreover, heightened CLEC11A expression positively correlated with immunomodulators, chemokines, and the infiltration of immune cells, especially M2 macrophages, in GC. Additionally, CLEC11A silencing suppressed GC cells proliferation, migration and invasion in vitro. Our results elucidate the functions of CLEC11A in GC, suggesting its potential as a valuable prognostic biomarker and therapeutic target for GC immunotherapy.",
                                                                                                      'PMID': '37597212',
                                                                                                      'DOI': '10.17305/bb.2023.9384',
                                                                                                      'Journal_Title': 'Biomolecules & biomedicine',
                                                                                                      'ISO_Abbreviation': 'Biomol Biomed',
                                                                                                      'Journal_Score': '1.78899082568807',
                                                                                                      'ISSN': '2831-090X',
                                                                                                      'ArticleDate': '2024-01-03',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/37597212/',
                                                                                                      'Level': 1}, {
                                                                                                      'Title': 'CLEC11A methylation is correlated to AML subtypes and cytogenetic risk factors but not patient demographics.',
                                                                                                      'Abstract': 'Acute myeloid leukemia (AML) is an aggressive and lethal cancer of the blood, which leads to the death of over 11,000 patients in the United States each year. Research on identifying, characterizing, and treating AML is crucial in the fight against this deadly disease. Recent studies have examined the role of CLEC11A in cancer, including AML. However, there have been conflicting reports related to tumor progression and survival. Because survival is based on a variety of factors, including classification of the tumor, genetic risk factors, and demographics, it is imperative that we determine what role CLEC11A may have in cancer survival. Therefore, utilizing data from the Genomic Data Commons, we analyzed CLEC11A methylation in 108 AML patients compared to FAB classification, cytogenetic risk factors, age, race, and gender. Our results show statistically significant correlations between methylation of CLEC11A and FAB classification as well as poor genetic risk factors. However, no difference was observed in CLEC11A methylation when compared to demographic data. Our results, matched with a known biological function of CLEC11A in early hematopoiesis, indicate that CLEC11A may be an important marker for AML diagnosis and prognosis and provide relevant data in the ongoing search for novel therapeutics to improve AML survival.',
                                                                                                      'PMID': '38466706',
                                                                                                      'DOI': '10.1371/journal.pone.0300477',
                                                                                                      'Journal_Title': 'PloS one',
                                                                                                      'ISO_Abbreviation': 'PLoS One',
                                                                                                      'Journal_Score': '3.06818181818182',
                                                                                                      'ISSN': '1932-6203',
                                                                                                      'ArticleDate': '2024-03-11',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/38466706/',
                                                                                                      'Level': 1}, {
                                                                                                      'Title': 'High Expression of CLEC11A Predicts Favorable Prognosis in Acute Myeloid Leukemia.',
                                                                                                      'Abstract': 'Acute myeloid leukemia (AML) is a heterogeneous disease of the hematopoietic system, for which identification of novel molecular\xa0markers is potentially important for clinical prognosis and is an urgent need for treatment optimization.',
                                                                                                      'PMID': '33747924',
                                                                                                      'DOI': '10.3389/fonc.2021.608932',
                                                                                                      'Journal_Title': 'Frontiers in oncology',
                                                                                                      'ISO_Abbreviation': 'Front Oncol',
                                                                                                      'Journal_Score': '2.65040650406504',
                                                                                                      'ISSN': '2234-943X',
                                                                                                      'ArticleDate': '2021-03-02',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/33747924/',
                                                                                                      'Level': 1}, {
                                                                                                      'Title': 'Hypermethylation of DNA Methylation Markers in Non-Cirrhotic Hepatocellular Carcinoma.',
                                                                                                      'Abstract': 'Aberrant DNA methylation changes have been reported to be associated with carcinogenesis in cirrhotic HCC, but DNA methylation patterns for these non-cirrhotic HCC cases were not examined. Therefore, we sought to investigate DNA methylation changes on non-cirrhotic HCC using reported promising DNA methylation markers (DMMs), including HOXA1, CLEC11A, AK055957, and TSPYL5, on 146 liver tissues using quantitative methylation-specific PCR and methylated DNA sequencing. We observed a high frequency of aberrant methylation changes in the four DMMs through both techniques in non-cirrhotic HCC compared to cirrhosis, hepatitis, and benign lesions (<i>p</i> < 0.05), suggesting that hypermethylation of these DMMs is specific to non-cirrhotic HCC development. Also, the combination of the four DMMs exhibited 78% sensitivity at 80% specificity with an AUC of 0.85 in discriminating non-cirrhotic HCC from hepatitis and benign lesions. In addition, HOXA1 showed a higher aberrant methylation percentage in non-cirrhotic HCC compared to cirrhotic HCC (43.3% versus 13.3%, <i>p</i> = 0.039), which was confirmed using multivariate linear regression (<i>p</i> < 0.05). In summary, we identified aberrant hypermethylation changes in HOXA1, CLEC11A, AK055957, and TSPYL5 in non-cirrhotic HCC tissues compared to cirrhosis, hepatitis, and benign lesions, providing information that could be used as potentially detectable biomarkers for these unusual HCC cases in clinical practice.',
                                                                                                      'PMID': '37835478',
                                                                                                      'DOI': '10.3390/cancers15194784',
                                                                                                      'Journal_Title': 'Cancers',
                                                                                                      'ISO_Abbreviation': 'Cancers (Basel)',
                                                                                                      'Journal_Score': '3.83529411764706',
                                                                                                      'ISSN': '2072-6694',
                                                                                                      'ArticleDate': '2023-09-28',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/37835478/',
                                                                                                      'Level': 1}, {
                                                                                                      'Title': 'Ultrasensitive and Quantitative DNA Methylation Detection Method Based on the MutS Protein.',
                                                                                                      'Abstract': "DNA methylation is closely related to cancer. It is generally accepted that DNA methylation detection is crucial in cancer diagnosis, prognosis, and treatment monitoring. Therefore, there is an urgent demand for developing a simple, rapid, highly sensitive, and highly specific methylation detection method to detect DNA methylation at specific sites quantitatively. In this work, we introduce a DNA methylation detection method based on MutS and methylation-specific PCR, named MutS-based methylation-specific PCR (MB-MSP), which has the advantages of simplicity, speed, high specificity, sensitivity, and broad applicability. Utilizing the MutS's ability to bind mismatched base pairs, we inhibit not only the amplification of unmethylated DNA but also nonspecific primer amplification. We achieved a detection sensitivity of 0.5% for the methylated genes of ACP1, CLEC11A, and SEPT9 by MB-MSP. It has a good linear relationship and a detection time of only 1.5 h. To validate the feasibility of the MB-MSP method in clinical application, we conducted methylation detection on plasma-circulating tumor DNA samples from 10 liver cancer patients and 5 healthy people, achieving a 100% accuracy rate. In conclusion, MB-MSP, as a novel and reliable DNA methylation detection tool, holds significant application value and potential for advancing early cancer diagnosis.",
                                                                                                      'PMID': '38078896',
                                                                                                      'DOI': '10.1021/acs.analchem.3c04013',
                                                                                                      'Journal_Title': 'Analytical chemistry',
                                                                                                      'ISO_Abbreviation': 'Anal Chem',
                                                                                                      'Journal_Score': '11.1',
                                                                                                      'ISSN': '1520-6882',
                                                                                                      'ArticleDate': '2023-12-11',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/38078896/',
                                                                                                      'Level': 2}, {
                                                                                                      'Title': 'Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia.',
                                                                                                      'Abstract': 'Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction.',
                                                                                                      'PMID': '39737198',
                                                                                                      'DOI': '10.3389/fimmu.2024.1494106',
                                                                                                      'Journal_Title': 'Frontiers in immunology',
                                                                                                      'ISO_Abbreviation': 'Front Immunol',
                                                                                                      'Journal_Score': '5.71875',
                                                                                                      'ISSN': '1664-3224',
                                                                                                      'ArticleDate': '2024-12-16',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/39737198/',
                                                                                                      'Level': 2}, {
                                                                                                      'Title': 'Identification and validation of a prognostic 8-gene signature for acute myeloid leukemia.',
                                                                                                      'Abstract': 'In the present study, we aimed to identify some genes closely related to AML prognosis and investigate their potential roles. RNA-seq data of AML samples were accessed from the TCGA database and then analyzed in the Wilcox test. AML survival-related genes were selected and an 8-gene signature-based risk score model was in turn constructed (including TET3, S100A4, BATF, CLEC11A, PTP4A3, SPATS2L, SDHA, and ATOX1 8 feature genes) using the multivariate Cox regression analysis. Kaplan-Meier analysis was performed on the 8 genes in the training set (<i>p</i>\u2009=\u20092.826e\u2009-\u200911) and the test set (<i>p</i>\u2009=\u20092.213e\u2009-\u20092), and there was a remarkable difference in survival between the high and low-risk samples. Meanwhile, ROC analysis was conducted and revealed the relative higher accuracy of the risk score model applied in both the training set (1-year AUC = 0.864; 3-year AUC = 0.85) and test set (1-year AUC = 0.685; 3-year AUC = 0.678). Our study helps to extend our knowledge of the potential methods for AML prognosis.HighlightsA prognostic 8-gene (including TET3, CLEC11A, ATOX1, S100A4, BATF, PTP4A3, SPATS2L and SDHA 8) signature for acute myeloid leukemia (AML) was identified and validated.The influence of the expression of single gene in the model on the survival risk of AML patients was confirmed and the risk rate of 8 single-gene was compared.',
                                                                                                      'PMID': '32268820',
                                                                                                      'DOI': '10.1080/10428194.2020.1742898',
                                                                                                      'Journal_Title': 'Leukemia & lymphoma',
                                                                                                      'ISO_Abbreviation': 'Leuk Lymphoma',
                                                                                                      'Journal_Score': '1.92156862745098',
                                                                                                      'ISSN': '1029-2403',
                                                                                                      'ArticleDate': '2020-04-08',
                                                                                                      'URL': 'https://pubmed.ncbi.nlm.nih.gov/32268820/',
                                                                                                      'Level': 1}]},
         {'biorxiv': [], 'medrxiv': [], 'pubmed': [{
                                                       'Title': 'Single-cell analysis reveals altered tumor microenvironments of relapse- and remission-associated pediatric acute myeloid leukemia.',
                                                       'Abstract': 'Acute myeloid leukemia (AML) microenvironment exhibits cellular and molecular differences among various subtypes. Here, we utilize single-cell RNA sequencing (scRNA-seq) to analyze pediatric AML bone marrow (BM) samples from diagnosis (Dx), end of induction (EOI), and relapse timepoints. Analysis of Dx, EOI scRNA-seq, and TARGET AML RNA-seq datasets reveals an AML blasts-associated 7-gene signature (CLEC11A, PRAME, AZU1, NREP, ARMH1, C1QBP, TRH), which we validate on independent datasets. The analysis reveals distinct clusters of Dx relapse- and continuous complete remission (CCR)-associated AML-blasts with differential expression of genes associated with survival. At Dx, relapse-associated samples have more exhausted T cells while CCR-associated samples have more inflammatory M1 macrophages. Post-therapy EOI residual blasts overexpress fatty acid oxidation, tumor growth, and stemness genes. Also, a post-therapy T-cell cluster associated with relapse samples exhibits downregulation of MHC Class I and T-cell regulatory genes. Altogether, this study deeply characterizes pediatric AML relapse- and CCR-associated samples to provide insights into the BM microenvironment landscape.',
                                                       'PMID': '37798266', 'DOI': '10.1038/s41467-023-41994-0',
                                                       'Journal_Title': 'Nature communications',
                                                       'ISO_Abbreviation': 'Nat Commun', 'Journal_Score': '13.5',
                                                       'ISSN': '2041-1723', 'ArticleDate': '2023-10-05',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/37798266/', 'Level': 2},
                                                   {
                                                       'Title': '<i>EGFR</i> Mutation-Harboring Lung Cancer Cells Produce CLEC11A with Endothelial Trophic and Tumor-Promoting Activities.',
                                                       'Abstract': 'The formation of new blood vessels in solid tumors is regulated by various endothelial trophic factors. We identified that CLEC11A, an extracellular C-type lectin, was over-expressed in lung cancer cell lines harboring mutated <i>EGFR</i>. CLEC11A expression was also frequently elevated in lung adenocarcinoma (LAC) tissues with <i>EGFR</i> mutation. CLEC11A-expressing H1299 cells formed larger tumors in nude mice than did the control cells. The CLEC11A-expressing tumors contained more CD31-positive cells, suggesting that they had a higher angiogenic activity. CLEC11A per se did not induce blood vessel formation, but enhanced angiogenesis triggered by VEGF-A or basic FGF in vivo. Additionally, the expression of small hairpin RNA against CLEC11A (shCLEC11A) in HCC827 LAC cells suppressed their tumorigenic ability. Purified CLEC11A exhibited a chemotactic ability, which is dependent on its integrin-binding RGD and LDT motifs, toward endothelial cells. This chemotactic activity was not affected by the presence of a VEGFR inhibitor. Conditioned medium produced by HCC827-shCLEC11A cells had diminished chemotactic ability toward endothelial cells. CLEC11A treatments increased the levels of active integrin β1 that were not associated with activation of focal adhesion kinases in endothelial cells. Our results indicated that CLEC11A was a factor of angiogenic potential and was involved in lung cancer tumorigenesis.',
                                                       'PMID': '35267664', 'DOI': '10.3390/cancers14051356',
                                                       'Journal_Title': 'Cancers',
                                                       'ISO_Abbreviation': 'Cancers (Basel)',
                                                       'Journal_Score': '3.83529411764706', 'ISSN': '2072-6694',
                                                       'ArticleDate': '2022-03-07',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/35267664/', 'Level': 1},
                                                   {
                                                       'Title': 'CLEC11A expression as a prognostic biomarker in correlation to immune cells of gastric cancer.',
                                                       'Abstract': "Gastric cancer (GC) is a prevalent malignant cancer characterized by a poor survival rate. The C-type lectin domain family 11 member A (CLEC11A) is part of the C-type lectin superfamily, and its dysregulation has been implicated in the progression of several cancers. The specific role of CLEC11A and its association with immune infiltration in GC, however, remains unclear. In this study, we employed The Cancer Genome Atlas (TCGA) database, Gene Expression Omnibus (GEO) database, Tumor IMmune Estimation Resource (TIMER) database, Gene Expression Profiling Interactive Analysis (GEPIA), UALCAN, Kaplan-Meier plotter databases, gene ontology (GO), Kyoto Encyclopedia of Genes and Genomes (KEGG), Gene Set Enrichment Analysis (GSEA), and the CIBERSORT algorithm to investigate CLEC11A expression, its prognostic significance, its association with tumor immune infiltration, and gene function enrichment in GC. We conducted western blotting, Cell Counting Kit-8 (CCK-8), wound healing, and transwell assays to validate CLEC11A's function. We found that CLEC11A expression was significantly elevated in GC when compared to adjacent non-tumor tissues. Elevated CLEC11A expression was strongly associated with poor survival outcomes and advanced clinicopathological stages. \xa0Moreover, heightened CLEC11A expression positively correlated with immunomodulators, chemokines, and the infiltration of immune cells, especially M2 macrophages, in GC. Additionally, CLEC11A silencing suppressed GC cells proliferation, migration and invasion in vitro. Our results elucidate the functions of CLEC11A in GC, suggesting its potential as a valuable prognostic biomarker and therapeutic target for GC immunotherapy.",
                                                       'PMID': '37597212', 'DOI': '10.17305/bb.2023.9384',
                                                       'Journal_Title': 'Biomolecules & biomedicine',
                                                       'ISO_Abbreviation': 'Biomol Biomed',
                                                       'Journal_Score': '1.78899082568807', 'ISSN': '2831-090X',
                                                       'ArticleDate': '2024-01-03',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/37597212/', 'Level': 1},
                                                   {
                                                       'Title': 'Single-cell transcriptomics reveals heterogeneity and prognostic markers of myeloid precursor cells in acute myeloid leukemia.',
                                                       'Abstract': 'Acute myeloid leukemia (AML) is a hematologic tumor with poor prognosis and significant clinical heterogeneity. By integrating transcriptomic data, single-cell RNA sequencing data and independently collected RNA sequencing data this study aims to identify key genes in AML and establish a prognostic assessment model to improve the accuracy of prognostic prediction.',
                                                       'PMID': '39737198', 'DOI': '10.3389/fimmu.2024.1494106',
                                                       'Journal_Title': 'Frontiers in immunology',
                                                       'ISO_Abbreviation': 'Front Immunol', 'Journal_Score': '5.71875',
                                                       'ISSN': '1664-3224', 'ArticleDate': '2024-12-16',
                                                       'URL': 'https://pubmed.ncbi.nlm.nih.gov/39737198/',
                                                       'Level': 2}]}]
